packages/create-cloudflare/src/helpers/command.ts

```diff
@@ -276,3 +300,28 @@ export const listAccounts = async () => {
 
 	return accounts;
 };
+
+/**
+ * Look up the latest release of workerd and use its date as the compatibility_date
+ * configuration value for wrangler.toml.
+ *
+ * If the look up fails then we fall back to a well known date.
+ *
+ * The date is extracted from the version number of the workerd package tagged as `latest`.
+ * The format of the version is `major.yyyymmdd.patch`.
+ *
+ * @returns The latest compatibility date for workerd in the form "YYYY-MM-DD"
+ */
+export async function getWorkerdCompatibilityDate() {
+	return runCommand("npm info workerd dist-tags.latest", {
+		silent: true,
+		captureOutput: true,
+		startText: "Retrieving current workerd compatibility date",
+		transform: (result) => {
+			const date = result.split(".")[1];
+			return `${date.slice(0, 4)}-${date.slice(4, 6)}-${date.slice(6, 8)}`;
+		},
+		fallback: () => "2023-05-18",
```

Comment:
Can this fallback to today/yesterday?

Reply:
Not really because that might be too recent for the current version of workerd.
Alternatively we could grab the most recent workerd date...
Happy for someone to follow up with a suitable alternative to this if desired.